includes/admin/settings/class-wc-settings-accounts.php

```diff
@@ -36,6 +36,8 @@ public function get_settings() {
 			$erasure_text = sprintf( '<a href="%s">%s</a>', esc_url( admin_url( 'tools.php?page=remove_personal_data' ) ), $erasure_text );
 		}
 
+		$tracking_info_text = sprintf( '<a href="%s">%s</a>','https://woocommerce.com/usage-tracking', esc_html__( 'Read more about what we collect' ) );
```

Comment:
Should we add `target="_blank"` to this link?

Reply:
Great idea!